src/dal_select2/static/autocomplete_light/select2.js

```diff
@@ -8,13 +8,24 @@
             var data_forward = {};
 
             for (var key in forward) {
+                var f = forward[key].split("->");
+                var src_name, dst_name;
+                if (f.length === 0) {
+                    continue;
+                } else if (f.length === 1) {
+                    src_name = f[0];
+                    dst_name = f[0];
+                } else {
+                    src_name = f.splice(0,1);
+                    dst_name = f.join("->");
+                }
```

Comment:
We can use JSON but we probably should not store it in an HTML attribute because it involves ugly escaping/unescaping code.
A possible way to handle this is something like that:
```html
<select><!--with some select2 and dal attrs-->
  <!-- Here we declare three forwards:
    - forward field_name value without renaming
    - forward field_name2 value as forward_name
    - forward constant 42 as const_forward_name
  -->
  <script type="text/forwardconfig">
    [
      {
      	"type": "field",
      	"src": "field_name"
      },
      {
      	"type": "field",
      	"src": "field_name2",
      	"dst": "forward_name"
      },
      {
        "type": "const",
        "value": 42
        "dst": "const_forward_name"
      }
    ]
  </script>
  <!-- Select options -->
</select>
```